app/models/planting_weather_log.rb

```diff
@@ -0,0 +1,50 @@
+# Defines a JSON data type as weather_log
+# See http://edgeguides.rubyonrails.org/active_record_postgresql.html#json for how to query it
+# See http://openweathermap.org/current for documentation
+
+# Example call: http://api.openweathermap.org/data/2.5/weather?q=London,uk&appid=df8abc00e3162fbd98bc48063cc6c4b5
+#  {
+#   "coord":{
+#     "lon":-0.13,
+#     "lat":51.51
+#   },
+#   "weather":[
+#     {
+#       "id":800,
+#       "main":"Clear",
+#       "description":"clear sky",
+#       "icon":"01d"
+#     }
+#   ],
+#   "base":"stations",
+#   "main":{
+#     "temp":281.83,
+#     "pressure":1025,
+#     "humidity":61,
+#     "temp_min":280.15,
+#     "temp_max":283.15
+#   },
+#   "visibility":10000,
+#   "wind":{
+#     "speed":5.7,
+#     "deg":300
+#   },
+#   "clouds":{
+#     "all":0
+#   },
+#   "dt":1491808800,
+#   "sys":{
+#     "type":1,
+#     "id":5091,
+#     "message":0.0097,
+#     "country":"GB",
+#     "sunrise":1491801287,
+#     "sunset":1491850179
+#   },
+#   "id":2643743,
+#   "name":"London",
+#   "cod":200
+# }
+class PlantingWeatherLog < ActiveRecord::Base
```

Comment:
I went for planting so you can answer things like "Why did my leeks grow better last year than this year? Oh, 27 days of 40 degree temperature in a row on last year's planting" more easily 

IE its simpler to do ```SELECT AVG(temp) FROM planting_weather_log WHERE planting_id = ?``` and only have data recorded for the duration of the planting; than it is to do

```SELECT AVG(temp) FROM garden_weather_log JOIN gardens JOIN plantings WHERE garden_weather_log.created_at BETWEEN planting.start and planting.end GROUP BY planting_id```
 x lots.


Plantings also have a start and end date usually (planting.active); while a Garden is forever; and at least one per user; even if they new do anything with the site. 
I suspect we'd run out of the free tier of API usage if we did that for everyone's gardens.

For dealing with plantings in the same garden/location/etc; we can probably cache the API requests at a city/geocoded location level.